pkgs/top-level/perl-packages.nix

```diff
@@ -3842,6 +3856,33 @@ let
     };
   };
 
+  CryptPassphrase = buildPerlPackage {
+    pname = "Crypt-Passphrase";
+    version = "0.003";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LE/LEONT/Crypt-Passphrase-0.003.tar.gz";
+      sha256 = "685aa090f8179a86d6896212ccf8ccfde7a79cce857199bb14e2277a10d240ad";
+    };
+    meta = {
+      description = "A module for managing passwords in a cryptographically agile manner";
+      license = with lib.licenses; [ artistic1 gpl1Plus ];
```

Comment:
Looks like release-20.09 doesn't have 276f91ec394f, so these commits will need to be modified by hand to reference `stdenv.lib` (or cherry-pick that commit...)

Reply:
Hmm it's kinda weird that my `nix-build` only caught this for one package oO
Will fix it :)